packages/pretty-format/src/plugins/ReactElement.js

```diff
@@ -70,7 +70,10 @@ function printProps(props, print, indent, colors, opts) {
 }
 
 const print = (
-  element: any,
+  // Disabling lint rule as we don't know inner type ahead of time.
+  /* eslint-disable flowtype/no-weak-types */
+  element: React$Element<any>,
```

Comment:
you can use `eslint-disable-next-line` to disable just one line.
also you can try using `React$Element<*>` instead of any